### ISSUE TITLE
How is the performance of using GT semantic mask directly?

### ISSUE THREAD

**Issue by zhangfuyang** [joel99/objectnav#1]
Hi Joel! Thanks for your awesome work! I am curious about the performance directly using GT semantic segmentation. Wondering if you could provide some info on this part.

Specifically, in Table 1 of the paper, I believe all the experiments are using RedNet (w/ or w/o fine-tuned). I wonder have you tried using gt directly? like "6-Action + Tether + GT semantic". BTW, is the pre-trained weight split_clamp.31.pth tuning with Rednet already?

I am trying to see the real challenge of this task. Is it the gap between predicted and gt segmentation or really the navigation?

Any comments would help. Thank you!

**Comment by joel99** [comment]
Hello, Table 3 compares the performance between GT and rednet segmentation. With GT, these agents get to, at best, ~55-60% Success. Navigation + segmentation are both challenging.
I believe `split_clamp.31.pth` is NOT tuned with RedNet. Hope this helps.

**Comment by zhangfuyang** [comment]
Thank you for your quick response!
